src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -924,6 +908,10 @@ export async function inlineSuggestionTextDocumentChangeHandler(
           suggestionId!,
           UserAction.ACCEPTED
         );
+        // Show training matches for the accepted suggestion.
```

Comment:
Moved those lines to `inlineSuggestionTextDocumentChangeHandler()` as content matches function should work when user accepted on a widget as well.